### ISSUE TITLE
MVX-Net batch norm in point_fusion.py 

### ISSUE THREAD

**Issue by og-lee** [open-mmlab/mmdetection3d#891]
Hello, I have some questions about MVX-Net point_fusion part. 

1.  I checked the feature values of the images in this part https://github.com/open-mmlab/mmdetection3d/blob/2df2b5bcb9a92ecae5ef153dd3815f434ed4da44/mmdet3d/models/fusion_layers/point_fusion.py#L224 and they were between -(e+19)  ~ (e+19) . Is this the value normal?  

2.  I also checked the parameter of the 'self.img_transform' and the running_var of batchnorm was all 'inf' like the picture below . 
This infinite variances made the 'img_pre_fuse' to have same vector for all the input. Is this an intended behavior?  I was curious because after training like this the evaluation score was similar to the paper. ![image](https://user-images.githubusercontent.com/78589530/131238737-17a63f5c-df47-4385-9b52-2b4cd480eeb3.png)

this image of the result value of 'self.img_tranfrom'
![image](https://user-images.githubusercontent.com/78589530/131238986-444dfd98-6d13-4880-9972-61b49ee21973.png)

**Comment by og-lee** [comment]
I just froze the backbone resnet and it seems okay now. 
I am not sure what was wrong with me. The backbone gradients all went to zero after few iteration.
